library/nginx

```diff
@@ -23,20 +23,22 @@ Architectures: amd64, arm32v6, arm64v8, i386, ppc64le, s390x
 GitCommit: 590f9ba27d6d11da346440682891bee6694245f5
 Directory: mainline/alpine-perl
 
-Tags: 1.12.2, stable, 1.12
+Tags: 1.14.0, stable, 1.12
```

Comment:
seems to be a `1.12` is left on all the tags :eyes: (`1.12-perl` `1.12-alpine` `1.12-alpine-perl`)